src/platforms/javascript/guides/capacitor/troubleshooting.mdx

```diff
@@ -5,6 +5,32 @@ keywords: ["minimum version", "deployment target"]
 sidebar_order: 1000
 ---
 
+## Different Versions for Sibling SDKs 
+
+To avoid any issues when installing packages or using the code on another machine, you must use the same version of Sentry Capacitor and @Sentry/Angular (or any other sibling SDK) that is referred by the peer dependency of Sentry Capacitor in your app.  
+
+Correct:
+
+```xml
+    "@sentry/angular": "7.13.0",
+    "@sentry/capacitor": "0.10.1"
+```
+
+Incorrect:
+
+```xml
+    "@sentry/angular": "^7.13.0",
+    "@sentry/capacitor": "^0.10.1"
+```
+
+If you see the following error during build time, the sibling SDKs installed in your app have mismatched versions:
+
+```js
+  node_modules/@sentry/types/types/globals.d.ts:2:11 - error TS2451: Cannot redeclare block-scoped variable '__DEBUG_BUILD__'.
+```
```

Comment:
Is this the error that someone is going to see that's going to make them come look at this documentation? If so, then this should be the first thing they see in this section. Right now, a user would have to read through the solution first before they'd ever see a reference to the error they're trying to get rid of.